pinot-query-planner/src/test/resources/queries/WindowFunctionPlans.json

```diff
@@ -14,6 +14,33 @@
           "\n"
         ]
       },
+      {
+        "description": "single empty OVER() count(*) only",
+        "sql": "EXPLAIN PLAN FOR SELECT COUNT(*) OVER() FROM a",
+        "output": [
+          "Execution Plan",
+          "\nLogicalProject(w0$o0=[$1])",
```

Comment:
This LogicalProject looks weird. I mean the `w0$o0=[$1]` part doesn't look familiar to me.

Reply:
Calcite creates such names for each of the window function columns generated as part of the query to ensure column uniqueness (from what I could understand). Do you think we need to rename these?

Reply:
fixed to generate better names ($0, $1, etc)